app/common/gristUrls.ts

```diff
@@ -703,6 +703,10 @@ export interface GristLoadConfig {
   // on this for custom domains, but set it generally for all pages.
   org?: string;
 
+  // True if there are no subdomains  or separate servers defined for
+  // the home servers or doc workers.
+  singleDomainSetup: boolean;
```

Comment:
Running this code through the grist-saas CI, it looks like the transition will be simpler if this field is optional like the others.

Reply:
Hm, I didn't want this to default to falsey, but I guess it won't for  `grist-core` as it will always be defined.

Okay, I'll allow `undefined` too.